msteams-platform/graph-permissions/resource-specific-consent.md

```diff
@@ -0,0 +1,172 @@
+|File.Read.Group|Open a file,get all text, and close the file in this team.|
+|File.Create.Group|Create or open a file in this team. Note: if the file already exists, its contents will be overwritten.|
+|File.Edit.Group|Update the name of a specified file in this team.|
+|File.Delete.Group|Delete a specified file in this team.|
+
+>[!NOTE]
+>Resource specific permissions are only available to Teams apps installed on the Teams client and are currently not part of Azure AD.
+
+### **Enabling RSC in your application** 
+
+## 1. Ensure that the Tenant admin has configured [group owner consent](/azure/active-directory/manage-apps/configure-user-consent#configure-group-owner-consent-to-apps-accessing-group-data) settings in Azure Active Directory (Azure AD)
+
+- **Disable or enable group owner consent from the Azure portal**  
+
+> [!div class="checklist"]
+>
+>- Sign in to the [Azure portal](https://portal.azure.com) as a [Global Administrator/Company Administrator](/azure/active-directory/users-groups-roles/directory-assign-admin-roles.md#global-administrator--company-administrator).  
+ > - Select **Azure Active Directory** =>**Enterprise applications** =>**User settings**.
+> - Enable, disable, or limit user consent with the control labeled **Users can consent to apps accessing company data for the groups they own** (This capability is enabled by default)
+
+![azure rsc configuration](../assets/images/azure-rsc-configuration.svg)
+
+| Value | Description|
+|--- | --- |
+|Yes | Enable group-specific consent for all group owners.|
+|No |Disable group-specific consent for all users.| 
+|Limited | Enable group-specific consent for members of a selected group.|
+
+- **Disable or enable group owner consent using PowerShell**. Please follow the steps outlined in  the [Configure group owner consent using PowerShell](/azure/active-directory/manage-apps/configure-user-consent#configure-group-owner-consent-using-powershell) documentation.
+
+## 2.  [Register your app using the Azure portal](/graph/auth-register-app-v2)
+
+>[!WARNING]
+>Do not register multiple Teams apps to the same Azure AD app id. The app id must be unique for each app. Attempts to install multiple apps to the same app id will fail.
+
+## 3. Check API permissions in the Azure portal
+
+Navigate to the **Home** => **App registrations** page and select your RSC app. Choose **API permissions** from the left nav bar and examine the list of configured permissions for your app. If your app will only make RSC Graph calls, delete all the permission on that page. If your app will also make non-RSC calls, keep those permissions as needed.
+
+>[!IMPORTANT]
+>The Azure portal cannot be used to request RSC permissions. RSC permissions are currently exclusive to Teams applications installed in the Teams client and are declared in the app manifest (JSON) file.
+
+## 4.  [Get an access token from the Microsoft Identity platform](/graph/auth-v2-user?view=graph-rest-1.0#3-get-a-token)
```

Comment:
Again I would say very explicitly you get this token the same way you would any other application permissions token. Unfortunately, there's quite a few ways to do that (with or without SDK, in multiple programming languages), so showing exactly how that's done would be a document in its own right.